controller/src/main/java/io/pravega/controller/store/stream/AbstractStreamMetadataStore.java

```diff
@@ -406,7 +409,7 @@ public OperationContext createContext(String scope, String name) {
     }
 
     @Override
-    public CompletableFuture<TxnStatus> sealTransaction(final String scopeName, final String streamName,
+    public CompletableFuture<Pair<TxnStatus, Integer>> sealTransaction(final String scopeName, final String streamName,
```

Comment:
Instead of relying on apache commons pair, could we use `AbstractMap.SimpleEntry` like we are doing elsewhere?